### ISSUE TITLE
Use Dask to parallelize tile & mosaic jobs for shorter runtimes

### ISSUE THREAD

**Issue by Zac-HD** [ANU-WALD/fire-data-processing#15]
It would be *really, really* nice if a one-core, 40-hour job could be a ten-core, four hour job.  This is possible with Dask.

Depends on #14 by requiring similar work in `onetile.py`, mostly for `get_fmc`.  Rprojection is a separate issue as it's more useful for other projects outside this repo.

Note that I have previously tried this and concluded that it's not worth the effort to debug, because we don't regenerate the archive often at all!  Also, test on small samples - it's easy to get it wrong and waste all but one core at substantial cost in KSU.
